src/Provider/Provider.Web/Views/Dashboard/_TransferredVacanciesAlert.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@using Esfa.Recruit.Vacancies.Client.Domain.Entities
+@model Esfa.Recruit.Provider.Web.ViewModels.Dashboard.TransferredVacanciesAlertViewModel
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <div class="info-summary">
+            <p class="govuk-body">
+                <span class="govuk-!-font-weight-bold">
+                    @Html.Raw(Model.LegalEntityNamesCaption)
+                </span>
+                <span asp-hide="@Model.HasTransfersToMultipleEmployers">
+                    has switched off your permission to create and manage vacancies on their behalf.
+                    Any vacancies created for this employer have been transferred to them
```

Comment:
Should we have a full stop at end of this sentence and the one below?

Reply:
Not according to the prototype. But I agree they need one. I'll add. 👍